ui/src/common/config.js

```diff
@@ -1,8 +1,8 @@
 import * as Sentry from '@sentry/browser';
 
-export function getConfigFor(configKey) {
+export function getConfigFor(configKey, notSetValue = null) {
   try {
-    return window.CONFIG[configKey];
+    return window.CONFIG[configKey] || notSetValue;
```

Comment:
`window.CONFIG[configKey] != null ? window.CONFIG[configKey] : notSetValue`

so that we don't return `notSetValue` when it's actually set to  `0` or `false` or `''`

Reply:
Also return `notSetValue` on error here at the end.